scripts/train_lm.py

```diff
@@ -144,15 +144,70 @@ def WriteMetaparameters(metaparameters, ngram_order, num_train_sets, out_file):
             i += 1
     f.close()
 
-def FormatMetaparameters(metaparameters):
+def FormatMetaparameters(metaparameters, num_train_sets):
```

Comment:
Thanks for this reminder. I'll modify it.

On Thu, Aug 18, 2016 at 10:27 PM, Wang Jian <notifications@github.com>
wrote:

> In scripts/train_lm.py
> <https://github.com/danpovey/pocolm/pull/52#discussion_r75419957>:
>
> > @@ -144,15 +144,70 @@ def WriteMetaparameters(metaparameters, ngram_order, num_train_sets, out_file):
> >              i += 1
> >      f.close()
> >
> > -def FormatMetaparameters(metaparameters):
> > +def FormatMetaparameters(metaparameters, num_train_sets):
>
> We only load the num_train_sets variable when args.bypass_metaparameter_optimization
> != None. If you need to use this value, please move the loading part of
> code outside the if condition.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/danpovey/pocolm/pull/52/files/87ca2779cd82ad774f490c940435f54fe5c991b7#r75419957>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ANVxSkWL3KRq_K2celZT3xl1dbNWv8P3ks5qhRR6gaJpZM4Jl6Bt>
> .
>



-- 
Ke Li
Dept. of Electrical and Computer Engineering
Johns Hopkins University
Email: kli26@jhu.edu